### ISSUE TITLE
Add scroll bars for lists

### ISSUE THREAD

**Issue by tkem** [tkem/mopidy-mobile#40]
Scrolling through large lists is a drag (pun intended).

Since ionic does not provide scroll bars or other options such as jump-bars quick navigation by itself, need to look for a third-party library or roll our own.

**Comment by tkem** [comment]
Since ionic itself does not provide these, setting this to "blocked" for now.

**Comment by notEvil** [comment]
After a quick test drive this is the first thing that keeps me from using mopidy-mobile, the search just doesn't replace a good scroll. Seems like Ionic provides scroll bars now.

**Comment by tkem** [comment]
@notEvil: Where did you get that info about Ionic providing scroll bars? Bear in mind that Mopidy-Mobile still uses Ionic v1, and that will probably not change in the immediate future due to lack of resources...

**Comment by notEvil** [comment]
True, v1 doesn't but later versions (if I'm not mistaken). Nevermind then
